software/messaging/src/main/java/org/apache/brooklyn/entity/zookeeper/ZooKeeperNode.java

```diff
@@ -46,7 +46,7 @@
 
     @SetFromFlag("downloadUrl")
     AttributeSensorAndConfigKey<String, String> DOWNLOAD_URL = ConfigKeys.newSensorAndConfigKeyWithDefault(SoftwareProcess.DOWNLOAD_URL,
-            "http://apache.fastbull.org/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
+            "http://apache.org/dist/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
```

Comment:
@neykov Mirrors come and go where as this site will always be present ... I think it is a smart default

The reason this has broke is because it previously relied on a thirdparty